### PR TITLE
update!: Split up Productivity Tools page

### LINKED ISSUE #2596: Split up Productivity Tools page
I feel like we've been treating this like a miscellaneous page. Why don't we just make all of these their own pages?

- Document Collaboration
- Office Suites
- Pastebins
- Language Tools

### PR DESCRIPTION
**Pull request by redoomed1**
Changes proposed in this PR:

- Split the current Productivity Tools page into separate pages
  - This PR is a continuation of #2598 and closes #2596.
  - No major changes to the content ported from the Productivity Tools page
  - Changes for the new recommendation pages
    - Morphed the current Productivity Tools page to the new Document Collaboration page, retaining the cover image and introduction
    - Added cover images and sidebar icons for the new pages (toggle the table below for details)
    - Added intros for the new Language Tools and Office Suites pages
    - Added sections for each new page in `tools.md` and `mkdocs.yml` and removed the Productivity Tools section from both
    - Updated image file paths
- Apply Wikipedia's style of bolding (recommended [here](https://github.com/privacyguides/privacyguides.org/pull/2620#discussion_r1648632857)) to tool recommendation pages where appropriate
- Reword the description for the Desktop Browsers page so that it's more consistent with [the one for the Mobile Browsers page](https://github.com/privacyguides/privacyguides.org/blob/main/docs/mobile-browsers.md?plain=1#L5)
- Update NextCloud internal link in the admonition on the Cloud Storage page
- Make minor wording changes otherwise not noted above

<details>
<summary>List of cover images and sidebar icons used</summary>

| Recommendation Page | Cover Image | Sidebar Icon |
|---|---|---|
| Document Collaboration | Same as the current one for Productivity Tools | [`account-group`](https://pictogrammers.com/library/mdi/icon/account-group/) |
| Language Tools | [Photo](https://unsplash.com/photos/choose-your-words-tiles-POMpXtcVYHo) by Brett Jordan (metaphorical) | [`alphabetical-variant`](https://pictogrammers.com/library/mdi/icon/alphabetical-variant/) |
| Office Suites | [Photo](https://unsplash.com/photos/multi-colored-pencils-on-pink-surface-vaTsR-ghLog) by Tamanna Rumee (metaphorical) | [`file-edit-outline`](https://pictogrammers.com/library/mdi/icon/file-edit-outline/) |
</details>

Here are the [steps](https://github.com/privacyguides/privacyguides.org/pull/2598#issuecomment-2143549648) I took to optimize the cover images. Any suggestions for alternative cover images or sidebar icons are welcome!

---

<summary>

- [x] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

File: docs/cloud.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ icon: material/file-cloud
 description: Many cloud storage providers require your trust that they will not look at your files. These are private alternatives!
 cover: cloud.webp
 ---
-Many cloud storage providers require your full trust that they will not look at your files. The alternatives listed below eliminate the need for trust by implementing secure E2EE.
+Many **cloud storage providers** require your full trust that they will not look at your files. The alternatives listed below eliminate the need for trust by implementing secure E2EE.
 
 If these alternatives do not fit your needs, we suggest you look into using encryption software like [Cryptomator](encryption.md#cryptomator-cloud) with another cloud provider. Using Cryptomator in conjunction with **any** cloud provider (including these) may be a good idea to reduce the risk of encryption flaws in a provider's native clients.
 
 <details class="TYPE" markdown>
 <summary>Looking for Nextcloud?</summary>
```
Comment (jonaharagon) [inline docs/cloud.md:13]: TYPE has to be replaced

**Review by jonaharagon — COMMENTED** [review]
Thanks for the PR!

Don't worry about changing anything for this PR, but in the future would you mind splitting something like this into smaller PRs please? Particularly at least these two changes could have been separate, since they are not related to the productivity page split:

- Apply Wikipedia's style of bolding (recommended https://github.com/privacyguides/privacyguides.org/pull/2620#discussion_r1648632857) to tool recommendation pages where appropriate
- Reword the description for the Desktop Browsers page so that it's more consistent with [the one for the Mobile Browsers page](https://github.com/privacyguides/privacyguides.org/blob/main/docs/mobile-browsers.md?plain=1#L5)

We (or maybe just I) prefer many smaller PRs over one larger one, to keep the logs nice, and to make reviews easier. Changes across 29 different files is a bit messy :)

**Comment by redoomed1** [comment]
> Changes across 29 different files is a bit messy :)

Oops, sorry about that :sweat_smile: Kind of got carried away here

> would you mind splitting something like this into smaller PRs please?

Of course I don't mind! I'll actually move the unrelated changes you listed into a separate PR anyway because the changes surrounding the Productivity Tools page are nontrivial and sizeable as is. (And I have some time on my hands.)

**Comment by redoomed1** [comment]
Superseded by #2661

**Pull request closed without merging**